src/main/java/com/goncharenko/currencyexchangerate/domain/User.java

```diff
@@ -0,0 +1,61 @@
+package com.goncharenko.currencyexchangerate.domain;
+
+import java.util.Objects;
+
+public class User {
```

Comment:
Lets use lombok @Getter @Setter instead of java plain code and @EqualsAndHashCode